Source/Plugins/Core/com.equella.core/js/tsrc/search/SearchPage.tsx

```diff
@@ -197,10 +197,19 @@ const SearchPage = ({ updateTemplate }: TemplateUpdateProps) => {
   }, [searchPageOptions]);
 
   useEffect(() => {
-    history.replace({
-      ...history.location,
-      state: { searchPageOptions, filterExpansion },
-    });
+    // If search page is open from a URL which has some query params, updating browser
+    // history for the initial search will produce wrong search option for the second
+    // render. This is because in the second render, we ignore the query param conversion
+    // as the history data has higher priority. So the end result is wrong search options
+    // are displayed in the page.
+    if (isInitialSearch && searchPageOptions === defaultSearchPageOptions) {
+      return;
+    } else {
+      history.replace({
+        ...history.location,
+        state: { searchPageOptions, filterExpansion },
+      });
+    }
```

Comment:
with the return, else is not needed
```suggestion
      return;
    }
    history.replace({
      ...history.location,
      state: { searchPageOptions, filterExpansion },
    });
```